GliaWidgets/Public/Glia/Glia+EngagementLauncher.swift

```diff
@@ -8,7 +8,23 @@ extension Glia {
     ///
     /// - Returns:
     ///   - `EngagementLauncher` instance.
-    public func getEngagementLauncher(queueIds: [String]?) -> EngagementLauncher {
-        .init(queueIds: queueIds)
+    public func getEngagementLauncher(queueIds: [String]?) throws -> EngagementLauncher {
+        guard let interactor else {
```

Comment:
I'd add configuration check before everything, because `interactor` is created on configuration step. So if the SDK is not configured, throwing `internalError` will confuse integrators.
```swift
guard let configuration = self.configuration else { throw GliaError.sdkIsNotConfigured }
```

Reply:
Totally agree.

Fixed https://github.com/salemove/ios-sdk-widgets/pull/1055/commits/35b99fb60b80b3827f49db315f47b1c344efdea9